sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/RecognizeCustomFormsAsyncWithSelectionMarks.java

```diff
@@ -43,7 +43,7 @@ public static void main(String[] args) throws IOException {
             .buildAsyncClient();
 
         // The form you are recognizing must be of the same type as the forms the custom model was trained on
-        File sourceFile = new File("../formrecognizer/azure-ai-formrecognizer/src/samples/resources/java/"
```

Comment:
what changed that now the samples don't need that `java` directory? or has this been wrong all the time?

Reply:
if it was wrong, don't the samples run in the CI? have they always been failing and no one noticed?

Reply:
Unfortunately, we currently don't have support on running samples in the CI. It is been failed all the time.

Reply:
oh no. how sad :( is this global for all Java?

Reply:
Yes. I talked to Alan about it. We will have a discussion about it.